agents/statsd/src/statsd_agent.h

```diff
@@ -284,9 +315,8 @@ class StatsDAgent: public std::enable_shared_from_this<StatsDAgent> {
   nsuv::ns_async env_msg_;
   TSQueue<std::tuple<SharedEnvInst, bool>> env_msg_q_;
 
-  StatsDTcp* tcp_;
+  StatsDConnection* connection_ = nullptr;
   std::unique_ptr<StatsDUdp> udp_;
-  std::unique_ptr<StatsDEndpoint> endpoint_;
   nsuv::ns_timer retry_timer_;
```

Comment:
you should be able to remove `udp_` and `retry_timer_`

Reply:
Whoops. Forgot those. Thanks.